functions/package.json

```diff
@@ -13,6 +13,7 @@
     "firebase": "^8.8.0",
     "firebase-admin": "^9.4.2",
     "firebase-functions": "^3.13.1",
+    "fs": "0.0.1-security",
```

Comment:
why is this necessary?

Reply:
Goood catch! Removed in https://github.com/sparkletown/sparkle/pull/2747/commits/f98865aa73e97ffb903a45cab387020444385cc4